plugin/src/main/java/io/fabric8/maven/plugin/ResourceMojo.java

```diff
@@ -453,6 +472,26 @@ private KubernetesList convertToOpenShiftResources(KubernetesList resources) {
                 }
             }
         }
+        if (isOpenShiftMode()) {
+            for (ImageConfiguration image : this.resolvedImages) {
+                String name = image.getName();
+                int idx = name.lastIndexOf(':');
+                if (idx < 0) {
+                    log.warn("No ':' in image name: " + name);
+                    continue;
+                }
+                String label = name.substring(idx + 1);
```

Comment:
Tip: For parsing image names you could also use the helper class [ImageName](https://github.com/fabric8io/docker-maven-plugin/blob/master/src/main/java/io/fabric8/maven/docker/util/ImageName.java) from d-m-p